libs/component-tests/src/main/java/org/codice/ddf/test/component/configurators/KarafOptions.java

```diff
@@ -0,0 +1,104 @@
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.editConfigurationFilePut;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.features;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.karafDistributionConfiguration;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.keepRuntimeFolder;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.logLevel;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.ops4j.pax.exam.ConfigurationManager;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.karaf.options.LogLevelOption;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KarafOptions implements ContainerOptions {
+    // TODO - Set RMI and other port configurations
+    private static final Logger LOGGER = LoggerFactory.getLogger(KarafOptions.class);
+
+    private static final String TEST_LOGLEVEL_PROPERTY = "logLevel";
+
+    private static final String TEST_DEBUG_PACKAGE_PROPERTY = "logPackages";
+
+    private static final String LOGGER_PREFIX = "log4j.logger.";
+
+    @Override
+    public Option get() {
+        return composite(karafDistributionConfiguration().frameworkUrl(maven().groupId(
+                "org.apache.karaf")
+                        .artifactId("apache-karaf")
+                        .version(getVersion())
+                        .type("zip"))
+                        .unpackDirectory(new File("target", "exam"))
+                        .useDeployFolder(false),
+                configureConsole().ignoreLocalConsole(),
+                logLevel().logLevel(LogLevelOption.LogLevel.WARN),
+                getLoggingOptions(),
+                getDebugOptions(),
+                getKeepRuntimeFolderOptions(),
+                getKarafFeatureOptions());
+    }
+
+    private String getVersion() {
+        ConfigurationManager cm = new ConfigurationManager();
+        return cm.getProperty("pax.exam.karaf.version");
+    }
+
+    private Option getLoggingOptions() {
+        String logLevel = System.getProperty(TEST_LOGLEVEL_PROPERTY, "DEBUG");
+
+        String packages = System.getProperty(TEST_DEBUG_PACKAGE_PROPERTY);
+
+        if (packages != null) {
+            List<Option> options = Arrays.stream(packages.split(","))
+                    .map(String::trim)
+                    .peek(p -> LOGGER.info("Enabling {} logs for {}", logLevel, p))
+                    .map(p -> editConfigurationFilePut("etc/org.ops4j.pax.logging.cfg",
+                            LOGGER_PREFIX + p,
+                            logLevel))
+                    .collect(Collectors.toList());
+            return composite(options.toArray(new Option[0]));
+        }
+
+        return null;
+    }
+
+    private Option getDebugOptions() {
+        return when(Boolean.getBoolean("debugEnabled")).useOptions(debugConfiguration());
```

Comment:
Should we change this option to be the same as the itest debug option `isDebugEnabled` or change the itests option. I think there should consistency for enabling test debugging.